submissions/karelskiy/HTML and CSS practice/style.css

```diff
@@ -0,0 +1,153 @@
+ body{
+    margin: 0;
+    font-family: 'Raleway', sans-serif;
+}
+
+.section{
+    position: absolute;
+    right:50px;
+    top: 3px;
+}
+.menu-btn{
+
+    display: block;
+    width: 50px;
+    height: 50px;
+    background-color: #fff;
+    border-radius: 50%;
+    position: relative;
+    
+}
+.menu-btn span,
+.menu-btn span::before,
+.menu-btn span::after {
+    position: absolute;
+    top: 50%;
+    margin-top : -1px;
+    left: 50%;
+    margin-left: -10px;
+    width: 20px;
+    height: 2px;
+    background-color: gray;
+}
+
+.menu-btn span::before,
+.menu-btn span::after{
+    content: '';
+    
+}
+
+.menu-btn span::before{
+    transform: translateY(-5px);
+}
+
+.menu-btn span::after{
+    transform: translateY(5px)
+}
+
+
+ .block {
+     display: none;
+     top: 50px;
+     right: 10px;
+    width: 270px;
+    height: 420px;
+    border: .8px solid #ccc;
+    padding: 0;
+    position: absolute;
+    box-shadow: 0 2px 10px rgba(0,0,0,.2);
+   }
+   .block::before, .block::after {
+    content: ''; 
+    position: absolute;
+    left: 195px; top: -20px;
+    border: 10px solid transparent;
+    transform: rotate(180deg);
+    border-top: 10px solid #ccc;
+   }
+
+   .block::after{
+    border-top: 10px solid white;
+    top: -19px; 
+    transform: rotate(180deg);
+    z-index: 9000;
+   }
+
+.open{
+    display: block;
+}
+
+#Doe{
```

Comment:
Avoid styling by `#id` and HTML tag.
Good practice to separate responsibilities: HTML tags for markup, Class - styling, #ID - JS.

Reply:
Thank for this tip